Source/Core/Core/HW/SI_DeviceGCController.cpp

```diff
@@ -26,6 +26,27 @@ CSIDevice_GCController::CSIDevice_GCController(SIDevices device, int _iDeviceNum
 {
 	// Dunno if we need to do this, game/lib should set it?
 	m_Mode                   = 0x03;
+
+	m_Calibrated = false;
+}
+
+void CSIDevice_GCController::GetOrigin()
+{
+	GCPadStatus pad_origin;
+	memset(&m_Origin, 0, sizeof(SOrigin));
+	memset(&pad_origin, 0, sizeof(GCPadStatus));
```

Comment:
This second memset isn't necessary, `GetPadStatus()` will zero out the one it returns.